tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
+  sourceMaps: "inline",
+  compact: false,
+  whitelist: [
```

Comment:
It might be overkill, but if we can't load babel-compiler as an isopacket, we could load the transform list as an isopacket. :P